README.md

```diff
@@ -1,51 +1,163 @@
 
-Makes web development less frustrating by selectively polyfilling just what the browser needs. Use it on your own site, or as a service.  For usage information see the [hosted service](https://cdn.polyfill.io), which formats and displays the service API documentation located in the [docs](docs/) folder.
+  * `PORT`: The port on which to listen for HTTP requests (default 3000).
+  * `NODE_ENV`: Name of environment.  We use `dev`, `prod`, `ci` or `qa`.  Just used for logging.
+  * `FASTLY_SERVICE_ID`, `FASTLY_API_KEY`: Used to fetch and render cache hit stats on the [usage] page of the hosted documentation.  If not specified, no stats will be shown.
+  * `PINGDOM_CHECK_ID`, `PINGDOM_API_KEY`, `PINGDOM_ACCOUNT`, `PINGDOM_USERNAME`, `PINGDOM_PASSWORD`: Used to fetch and render uptime and response time stats on the [usage] page of the hosted documentation.  If not specified, no stats will be shown.
+  * `GRAPHITE_HOST`: Host to which to send Carbon metrics.  If not set, no metrics will be sent.
+  * `GRAPHITE_PORT`: Port on the `GRAPHITE_HOST` to which to send Carbon metrics (default 2002).
+  * `SAUCE_USER_NAME` and `SAUCE_API_KEY`: [Sauce Labs][sauce] credentials for grunt test tasks (not used by the service itself)
+  * `ENABLE_ACCESS_LOG`: Any truthy value will enable writing an HTTP access log from Node. Useful if we are not running node behind a routing layer like nginx or heroku.
 
-## Installing as a service
 
-1. Install [git](http://git-scm.com/downloads) and [Node](http://nodejs.org) on your system
-2. Clone this repository to your system (`git clone git@github.com:Financial-Times/polyfill-service.git`)
-3. Run `npm install` (this will also run `grunt build` which you will need to do again if you edit any polyfills)
+Testing
+-------
 
-To run the app for **development**:
+The tests are split into for the service and tests for the polyfills. The polyfill tests require `SAUCE_USER_NAME` and `SAUCE_API_KEY` to be configured, view the [configuration](#configuration) section for more information.
 
-Run `grunt dev` from the root of the working tree.  This will watch your filesystem and automatically rebuild sources and restart if you make any changes to any of the app source code or polyfills.  If you change the docs, library or service (`docs`, `lib` and `service` directories) the service will be restarted.  If you change the polyfills or polyfill configs (`polyfills` directory), the polyfill sources will be recompiled before the service restarts, which makes the restart slightly slower.
+```sh
+grunt test           # run service tests and polyfill tests on a small set of browsers
+grunt simplemocha    # run the service tests
+grunt ci             # run the service tests and polyfills tests on a large set of browsers
+```
+
+We run the tests on CI, we can see the [results on CircleCI][ci].  `grunt ci` must pass before we merge a pull request.
```

Comment:
_The continuous integration tests are run on Circle CI where we can see the results._

How about that?